volume-providers/vpc/vpcclient/instances/iks_attach_volume.go

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018, 2019 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package instances
+
+import (
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/utils"
+	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/client"
+	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/models"
+	"go.uber.org/zap"
+	"time"
+)
+
+// AttachVolume attached volume to instances with givne volume attachment details
+func (vs *IKSVolumeAttachService) AttachVolume(volumeAttachmentTemplate *models.VolumeAttachment, ctxLogger *zap.Logger) (*models.VolumeAttachment, error) {
+	defer util.TimeTracker("IKS AttachVolume", time.Now())
+
+	operation := &client.Operation{
+		Name:   "AttachVolume",
+		Method: "POST",
+	}
+	operation.PathPattern = vs.pathPrefix + "createAttachment"
+
+	var volumeAttachment models.VolumeAttachment
+	apiErr := vs.receiverError
+
+	request := vs.client.NewRequest(operation)
+
+	request = request.SetQueryValue(IksClusterQueryKey, *volumeAttachmentTemplate.ClusterID)
+	request = request.SetQueryValue(IksWorkerQueryKey, *volumeAttachmentTemplate.InstanceID)
+	vol := *volumeAttachmentTemplate.Volume
+	request = request.SetQueryValue(IksVolumeQueryKey, vol.ID)
```

Comment:
It should not be part of QueryString , it should be part of body as per POST api standard